### PR TITLE
Adding google calendar iframe to houses page

### PR DESCRIPTION
**Pull request by JKThanassi**
Adds google calendar to houses page. Calendar links can be added through a form in the admin page for houses. Some style things need to be fixed as well.

### CODE REVIEW

**Comment by tmwbook** [comment]
I had been thinking this over and what the potential security risks could be at play. Right now we are just letting a user input an arbitrary URL and then rendering an <iframe>.  This seems like it could be used for other purposes.  We should try to validate or restrict the url in some way to prevent this.

**Comment by JKThanassi** [comment]
Sounds like a good idea, that was an oversight on my part. Would checking to see if the beginning of the link contains `calendar.google.com` be enough?

File: houses/forms.py [hunk 1]
```diff
@@ -7,3 +8,10 @@ class HouseForm(forms.ModelForm):
     class Meta:
         model = Fraternity
         fields = '__all__'
+
+    def clean_calendar_link(self):
+        data = self.cleaned_data['calendar_link']
+        if not re.match("^https://calendar.google.com/calendar/embed\?src=(.*)", data):
```
Comment (JKThanassi) [inline houses/forms.py]: @tmwbook does this address your concerns?

Comment (tmwbook) [reply houses/forms.py]: I am in the process of writing unit tests to see if this is verbose enough.

**Comment by JKThanassi** [comment]
@tmwbook good to merge?

**Pull request merged**